apps/api/src/planning-session/planning-session.service.ts

```diff
@@ -171,6 +179,7 @@ export class PlanningSessionService {
     await this.planningSessionRepo.save({
       ...planningSession,
       occupation,
+      updatedAt: new Date(), // Manually updating as TypeORM does not update this field when relations are updated;
```

Comment:
I might be missing something, but I don't see the updatedAt column on the planning session entity. 
Typeorm has a specific decorator @UpdateDateColumn for to handle this automatically.

Reply:
That makes sense!